### ISSUE TITLE
`.astify()` not working with CONVERT

### ISSUE THREAD

**Issue by Val-istar-Guo** [taozhi8833998/node-sql-parser#920]
**Describe the bug**

Run it in https://npm.runkit.com/node-sql-parser

```
const { Parser } = require('node-sql-parser');
const parser = new Parser();
const ast = parser.astify(`select TYPE,taxpayer_Type,CONVERT(tax_Amount, DECIMAL(12,2)) AS tax_amount,CAST(tax_currency AS DECIMAL(12,2))  tax_currency from rs_order_tax where billno="{{billno}}" and Business_Type="order";
`);


console.log(parser.sqlify(ast))
```

Will get a string: 
```sql
SELECT `TYPE`, `taxpayer_Type`, CONVERT(`tax_Amount`, DECIMAL) AS `tax_amount`, CAST(`tax_currency` AS DECIMAL(12, 2)) AS `tax_currency` FROM `rs_order_tax` WHERE `billno` = '{{billno}}' AND `Business_Type` = 'order'
```

The `(12, 2)` is missing

**Database Engine**
MySQL

**Expected behavior**
`.astify()` should not change the SQL meaning

**Screenshots**
![image](https://user-images.githubusercontent.com/12405937/151115918-a5ce8c48-073b-42ea-b9c5-e8d7786d35d1.png)

**Comment by taozhi8833998** [comment]
@Val-istar-Guo I will publish a new version this week. Happy Spring Festival!

**Comment by Val-istar-Guo** [comment]
新年快乐～

**Comment by taozhi8833998** [comment]
@Val-istar-Guo I have published a new version `4.1.0`, try it.
